pkgs/by-name/jn/jnv/package.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, autoconf
+, automake
+, clang
+, libclang
+, libtool
+}:
+rustPlatform.buildRustPackage rec
+{
+  pname = "jnv";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "ynqa";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-LN42UyXOStwgj/bbHl2MK1jTsDACTjDo+QaoZ9FzurA=";
+  };
+
+  cargoHash = "sha256-ZCYDlvKSuEONPSz/TS00mYolPnMYxsOwE9a0vK36UQQ=";
+
+  nativeBuildInputs = [
+    autoconf
+    automake
+    clang
+    libclang
+    libtool
+  ];
+
+  cargoPatches = [
+    ./cargo-lock.patch
+  ];
+
+  LIBCLANG_PATH = "${libclang.lib}/lib";
+
+  meta = with lib; {
+    description = "interactive JSON filter using jq";
+    mainProgram = "jnv";
+    homepage = "https://github.com/ynqa/jnv";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ nealfennimore ];
+    platforms = platforms.linux;
```

Comment:
what about Darwin (macOS)? `platforms.linux ++ platforms.darwin`
Maybe `platforms.unix`?

Reply:
Updated to `unix`